extra-utils/wallcolle-make/autobuild/build

```diff
@@ -0,0 +1,3 @@
+abinfo "Building \"wallcolle-make\"..."
```

Comment:
I mean, you're clearly _installing_ it...

Reply:
Aha, wording...